### PR TITLE
nami-rkuji

### PR DESCRIPTION
**Pull request by enriquesouza**
nami + rkuji

Receive Nami OK
Send Nami OK
https://finder.kujira.network/kaiyo-1/tx/425C63C685F6D707BCC6751EC4E4EFE91CD5493B3F02677C55A61A2DEBC3D1D7

Receive rKuji Not tested I don't know how to get any. -> Should work.
https://github.com/cosmos/chain-registry/blob/master/kujira/assetlist.json#L2662

image

### CODE REVIEW

**Review by johnnyluo — APPROVED** [review]
LGTM

File: VultisigApp/VultisigApp/Stores/TokensStore.swift [hunk 1]
```diff
@@ -607,6 +607,14 @@ class TokensStore {
             CoinMeta(chain: Chain.kujira, ticker: "USK", logo: "usk", decimals: 6, priceProviderId: "usk", contractAddress: "factory/kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7/uusk", isNativeToken: false)
         }
         
+        static var kujiraNami: CoinMeta {
+            CoinMeta(chain: Chain.kujira, ticker: "NAMI", logo: "nami", decimals: 6, priceProviderId: "nami-protocol", contractAddress: "factory/kujira13x2l25mpkhwnwcwdzzd34cr8fyht9jlj7xu9g4uffe36g3fmln8qkvm3qn/unami", isNativeToken: false)
+        }
+            
+        static var kujiraRKuji: CoinMeta {
+            CoinMeta(chain: Chain.kujira, ticker: "rKUJI", logo: "rkuji", decimals: 6, priceProviderId: "rkuji", contractAddress: "factory/kujira1tsekaqv9vmem0zwskmf90gpf0twl6k57e8vdnq/urkuji", isNativeToken: false)
```
[the remark this answers is not in the document]
Comment (enriquesouza) [reply VultisigApp/VultisigApp/Stores/TokensStore.swift:615]: Thanks @SxMShaDoW

[the remark this answers is not in the document]
Comment (enriquesouza) [reply VultisigApp/VultisigApp/Stores/TokensStore.swift:615]: https://github.com/vultisig/vultisig-ios/pull/1662

**Comment by kulivoko** [comment]
but even with rKuji visible in my vultisig wallet it does not count for vult airdrop :(

**Comment by kulivoko** [comment]
> nami + rkuji
> 
> Receive Nami OK Send Nami OK https://finder.kujira.network/kaiyo-1/tx/425C63C685F6D707BCC6751EC4E4EFE91CD5493B3F02677C55A61A2DEBC3D1D7
> 
> Receive rKuji Not tested I don't know how to get any. -> Should work. https://github.com/cosmos/chain-registry/blob/master/kujira/assetlist.json#L2662
> 
> image

but even with rKuji visible in my vultisig wallet it does not count for vult airdrop :(

**Pull request merged**